src/main/java/io/goobox/sync/storj/App.java

```diff
@@ -146,6 +153,10 @@ public OverlayHelper getOverlayHelper() {
         return overlayHelper;
     }
 
+    public StorjExecutorService getStorjExecutorService() {
```

Comment:
Is this getter used anywhere? If not, we should not have it.

Reply:
okay